test/libyul/yulOptimizerTests/fakeStackLimitEvader/function_arg.yul

```diff
@@ -0,0 +1,20 @@
+{
+	mstore(0x40, memoryguard(0))
+	let $x := 0
+	sstore(0, $x)
+	function h($hx) -> y {
+		y := $hx
+	}
+	sstore(1, h(32))
+}
+// ----
+// step: fakeStackLimitEvader
+//
+// {
+//     mstore(0x40, memoryguard(0x40))
+//     mstore(0x20, 0)
+//     sstore(0, mload(0x20))
+//     function h($hx) -> y
+//     { y := $hx }
```

Comment:
``$hx`` is marked for being escalated and ``0x00`` is actually reserved for it, but we cannot move it, since it's a function argument, so the entire function is left untouched (so referencing ``$hx`` will still be a "fake" stack error).